packages/esm-ward-app/src/beds/occupied-bed.test.tsx

```diff
@@ -5,33 +5,26 @@ import { bedLayoutToBed, filterBeds } from '../ward-view/ward-view.resource';
 import { getDefaultsFromConfigSchema, useConfig } from '@openmrs/esm-framework';
 import { configSchema, defaultPatientCardElementConfig } from '../config-schema';
 import { mockAdmissionLocation } from '../../../../__mocks__/wards.mock';
-import { mockAdmittedPatient } from '../../../../__mocks__/patient.mock';
+import { mockPatientAlice, mockPatientBrian } from '__mocks__';
 
 const defaultConfig = getDefaultsFromConfigSchema(configSchema);
 
 jest.mocked(useConfig).mockReturnValue(defaultConfig);
 
 const mockBedLayouts = filterBeds(mockAdmissionLocation);
-
 const mockBedToUse = mockBedLayouts[0];
-jest.replaceProperty(mockBedToUse.patients[0].person, 'preferredName', {
-  uuid: '',
-  givenName: 'Alice',
-  familyName: 'Johnson',
-});
```

Comment:
See above, I just added the preferred name directly to the mock, and didn't seem to cause issues.